corehq/apps/user_importer/tests/test_importer.py

```diff
@@ -2066,7 +2066,55 @@ def test_invite_location_add(self):
             self.upload_record.pk,
             True
         )
-        self.assertEqual(getattr(self.user_invite.primary_location, 'location_id', None), self.loc1._id)
+        self.assertEqual(self.user_invite.primary_location, self.loc1)
+        self.assertListEqual(
+            list(self.user_invite.assigned_locations.values_list('location_id', flat=True)),
+            [self.loc1._id]
+        )
+
+        import_users_and_groups(
+            self.domain.name,
+            [self._get_invited_spec(location_code=[])],
+            [],
+            self.uploading_user.get_id,
+            self.upload_record.pk,
+            True
+        )
+        self.assertIsNone(self.user_invite.primary_location)
+        self.assertListEqual(
+            list(self.user_invite.assigned_locations.values_list('location_id', flat=True)),
+            []
+        )
+
+    def test_invite_location_column_missing(self):
+        self.setup_locations()
+        import_users_and_groups(
+            self.domain.name,
+            [self._get_invited_spec(location_code=[a.site_code for a in [self.loc1]])],
```

Comment:
nit: I know you are just keeping this how it was done previously, but feels unnecessarily complicated when using one location.
```suggestion
            [self._get_invited_spec(location_code=[self.loc1.site_code])],
```

Reply:
https://github.com/dimagi/commcare-hq/pull/36931/commits/b8215c25ff3a0d87610f6a29b69ae88c01bb5375